.github/ISSUE_TEMPLATE/beta_feature.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Beta Feature Requirements
+about: Requirements for promoting a feature to beta
+---
+
+This page lists the requirements for promoting a feature to beta. Promotion to beta includes all requirements for promotion to alpha. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+
+**Promotion to Alpha:**
+
+**Relevant documentation:**
+
+```
+```
+
+--- 
+
+### Requirements: 
+
+**Design**
+
+- [ ] Design doc has been approved by relevant work group leads
+
+**Docs** 
+
+- [ ] Documentation on istio.io includes performance expectations; may have caveats. 
+- [ ] Documentation on istio.io includes samples/tutorials. 
+- [ ] Documentation on istio.io includes appropriate glossary entries. 
+- [ ] All new documentation containing user actions includes istio.io tests.
+- [ ] Release notes have been added. 
+- [ ] Upgrade notes have been added. 
+
+**Tests**
+
+- [ ] Integration tests cover feature edge cases
+- [ ] End-to-end tests cover samples/tutorials
+- [ ] Fixed issues have tests to prevent regressions
+- [ ] Stability/stress test suite includes coverage for the feature.
+
+**Performance**
+
+- [ ] Feature has baseline performance tests
+
+**API**
+
+- [ ] TOC has reviewed the API and determined it to be complete. 
+
+**CLI**
+
+- [ ] Any necessary CLI commands have been implemented and are complete. 
```

Comment:
Tooling to debug the feature

Reply:
Hmmm.

Any necessary tooling to configure/use/debug/deploy feature has been implemented and is complete?

This could be isioctl commands, debug features, deployment methods, or anything else. In the community PR, it was agreed on that these would be CLI commands, but it could be anything. We might want to expand this beyond CLI as well.